group.yml

```diff
@@ -1,17 +1,53 @@
 arches:
 - x86_64
+
+# switches for experimental Doozer features
+doozer_feature_gates:
+  # odcs_enabled: whether to enable Doozer ODCS support (without explicitly specifying --odcs-mode option)
+  # See https://issues.redhat.com/browse/ART-128 for more information about ODCS mode.
+  odcs_enabled: true
+  # odcs_aggressive: true to build an image with ODCS auto mode even if its image config doesn't have `odcs` field defined
+  odcs_aggressive: true
+
+advisories:
+  rpm: 0
+  image: 0
+  security: 0
+
+build_profiles:
+  image:
+    unsigned:
+      targets:
+      - rhaos-{MAJOR}.{MINOR}-rhel-7-containers-candidate
+      signing_intent: unsigned
+      repo_type: unsigned
+    signed:
+      targets:
+      - rhaos-{MAJOR}.{MINOR}-rhel-7-signed-containers-candidate
+      signing_intent: release
+      repo_type: signed
+  rpm:
+    default:
+      targets:
+      - rhaos-{MAJOR}.{MINOR}-rhel-7-candidate
+
+default_image_build_profile: unsigned
+default_rpm_build_profile: default
+
 branch: rhaos-{MAJOR}.{MINOR}-rhel-7
+default_image_build_method: imagebuilder
```

Comment:
i would leave this out. it's not _necessary_ and might introduce incompatibilities we never dealt with in 3.7.

Reply:
Removed